tensorrt_llm/_torch/visual_gen/cute_dsl_kernels/helpers/fmha_helpers.py

```
@@ -0,0 +1,1176 @@
+        :type index_qk: cute.Tensor
+        :param seqlen_k: Key sequence length for attention computation.
+        :type seqlen_k: Int32
+        :param seqlen_q: Query sequence length for attention computation.
+        :type seqlen_q: Optional[int]
+        :param window_size_left: Left-side sliding window size for attention masking.
+        :type window_size_left: Optional[int]
+        :param window_size_right: Right-side sliding window size for attention masking.
+        :type window_size_right: Optional[int]
+        """
+
+        tidx, tidy, tidx = cute.arch.thread_idx()
```

Ruff (0.15.20)

[warning] Unpacked variable `tidy` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)